opennms-container/minion/CONFD_README.md

```diff
@@ -161,3 +161,33 @@ system:
 ```
 Config specified will be written to `etc/confd.system.properties` which gets automatically appended to `etc/system.properties`. Additionally, provided the
 `jaeger-agent-host` key is specified, `etc/featuresBoot.d/jaeger.boot` will also be updated.
+
+### Karaf Properties
+```
+---
+karaf:
+    shell:
+        ssh:
+            host: "0.0.0.0"
+            port: 8201
+    management:
+        rmi:
+            registry:
+                host: "127.0.0.1"
+                port: 1299
+            server:
+                host: "127.0.0.1"
+                port: 45444
+```
+Config specified will be written to:
+- `etc/org.apache.karaf.shell.cfg` for content under `shell`.
+- `etc/org.apache.karaf.management.cfg` for content under `management`.
+
+### Jolokia Properties
+```
+---
+jolokia:
+    web:
+        port: 8181
```

Comment:
Can we add the host as well? We should be able to bind this to the loopback.

`org.ops4j.pax.web.listening.addresses` may work - see https://ops4j1.jira.com/wiki/spaces/paxweb/pages/12059275/Basic+Configuration

Reply:
Added; done.